SchleswigGottorf/GotD781.xml

```diff
@@ -28,21 +28,382 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                    <textLang mainLang="am" otherLangs="gez"/>
+                  <incipit xml:lang="gez">
+                    <hi rend="rubr">አገባብ፡</hi> የሚሉ፡ አገባብ፡ የሚባሉ፡ የሟሰኙ፡
+                    ፫ናቸው፡ እርባታ፡ የሚሉ፡ እርባታ፡ የሚባሉ፡ እርባታ፡ የሚያሰኙ፡ ፫ናቸው፡ ፫ቱ፡ ምንምን፡ ናቸው፡ ቢሉ፡ ባሉ፡ እንደሆነ፡
+                    </incipit>
+               </msItem>
+
+               <msItem xml:id="ms_i1.2.5">
+                 <locus from="73rb" to="76rb"/>
+                 <title type="complete">Grammatical explanation on numerals (አኃዝ፡)</title>
+                 <textLang mainLang="am" otherLangs="gez"/>
+               <incipit xml:lang="gez">
+                </incipit>
+            </msItem>
+
+            <msItem xml:id="ms_i1.2.6">
+              <locus from="76rb" to="78vb"/>
+              <title type="incomplete">Unidentified text</title>
+              <textLang mainLang="am" otherLangs="gez"/>
+            <incipit xml:lang="gez">ሰሶበ፡ ሰምአ፡ ዳዊት፡ ደንገጸት፡ ነፍሱ፡ ወነፍሰ፡ ኵሉ፡ ሕዝቡ፡ ፮መቶ፡ ሰብአ፡
+              ገባዖንን፡ በኰክሕ፡ ወሬሞን፡ ዳዊት፡ ፫መቶ፡ ሆኖ፡ በሐቅለ፡ ኤደልም፡ ሳኡል፡ ፮ሽህ፡ ደቂቀ፡ ኤፍሬም፡ ሲል፡
+             </incipit>
+         </msItem>
+             </msItem>
+              </msItem>
+
                     </msContents>
-                   <physDesc>
-                     <objectDesc form="Codex">
-                        <supportDesc>
-                           <support>
-                              <material key="parchment"/>
-                           </support>
-                            <foliation>The manuscript is not foliated.</foliation>
+                    <physDesc>
+                       <objectDesc form="Codex">
+                          <supportDesc>
+                             <support>
+                                <material key="parchment"/>
+                                </support>
+                             <extent>
+                               <measure unit="leaf">77</measure>
+                               <measure unit="leaf" type="blank">2</measure>
+                                <locus target="#2r #41v"/>
+                                <measure unit="quire">11 (A+10)</measure>
+
+                                <dimensions type="outer">
+                                   <height unit="mm">180</height>
+                                   <width unit="mm">130</width>
+                                   <depth unit="mm">51</depth>
+                                </dimensions>
+                             </extent>
+                             <foliation>No foliation</foliation>
+                             <collation>
+                               <signatures>
+                                  <note>Undecorated quire marks are written on the first page of some quires.</note>
+                               </signatures>
+                               <list>
+                                  <item xml:id="q1" n="A">
+                                      <locus from="1r" to="2v"/>
```

Comment:
`dim `is missing
`locus target` should go away


should be

```
<item xml:id="q1" n="A">
                                      <dim unit="leaf">1</dim>
                                      <locus from="1r" to="2v"/>
                                       s.l. 1, no stub
                                     </item>
```

Reply:
done